lib/src/ArrayUtils.ts

```diff
@@ -167,4 +168,13 @@ export class ArrayUtils {
         return result;
     }
 
+    public static hasDuplicates<V extends Comparable>(list: ReadonlyArray<V>): boolean {
```

Comment:
est-il utilisé ? (live tu me dit que non)
Si oui, tu peux remplacer la conversion en set par une double boucle qui s'arrête au premier dupliqué, ça gagnera du temps vs. convertir toute la liste et voir ensuite si on avait des dupliqués
Si non, kill it with fire